### PR TITLE
Initial draft of changelog for 0.12.0

### PR DESCRIPTION
**Pull request by j9ac9k**
Changelog for 0.12.0 release, leaving in draft form in case we want to leave some last minute changes...

### CODE REVIEW

**Comment by j9ac9k** [comment]
I'm thinking #1609 should be fixed before we release.   If anyone thinks another issue/PR should be resolved before we roll versions let me know.

File: CHANGELOG [hunk 1]
```diff
@@ -1,9 +1,71 @@
...
+  Bug Fixes:
+    - #1487 Fix InfiniteLabel object has not attribute 'updateText'
+    - #1491 LinearRegionItem would break with setSpan
+    - #1496 enableMenu setting now preserved when passing ViewBox to PlotItem
+    - #1498 PlotDataItem now signals on setPos()
+    - #1500 AlignCenter should have been AlignHCenter
+    - #1506 Fix "camerPosition" typo in GLViewWidget
+    - #1510 Fix RemoteGraphicsView mouse interactions on Qt 5.12
+    - #1517 Fix RemoteSpeedTest Shutdown Errors
+    - #1528 Support suffix for int parameters
```
Comment (NilsNemitz) [inline CHANGELOG:51]: ...in SpinBox ?
((and was that a bug or an added feature?))

File: CHANGELOG [hunk 2]
```diff
@@ -1,9 +1,71 @@
+pyqtgraph-0.12.0
+
+  Deprecations:
+    - Qt < 5.12, Python < 3.6, and numpy < 1.17 not supported
```
Comment (NilsNemitz) [inline CHANGELOG:4]: "not supported" --> "are no longer supported" ?

File: CHANGELOG (hunk 1 above)
Comment (j9ac9k) [reply CHANGELOG:51]: A little of both, it worked for floats I think, but not ints, ... so I would say this likely belongs in bug-fix territory

**Comment by ksunden** [comment]
LGTM

**Pull request merged**